bouncer/shared/create_and_delete_multiple_orders.ts

```diff
@@ -1,15 +1,39 @@
+import { ApiPromise } from '@polkadot/api';
 import { Keyring } from '../polkadot/keyring';
 import { handleSubstrateError, lpMutex } from '../shared/utils';
 import { getChainflipApi, observeEvent } from './utils/substrate';
 import { limitOrder } from './limit_order';
+import { rangeOrder } from './range_order';
+
+async function countOpenOrders(
+  baseAsset: string,
+  quoteAsset: string,
+  lp: string,
+  chainflip: ApiPromise,
+) {
+  const orders = await chainflip.rpc('cf_pool_orders', baseAsset, quoteAsset, lp);
+  if (!orders) {
+    throw Error('Rpc cf_pool_orders returned undefined');
+  }
+  let openOrders = 0;
+
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.asks.length || 0;
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.limit_orders.bids.length || 0;
+  // @ts-expect-error limit_orders does not exist on type AnyJson
+  openOrders += orders?.range_orders.length || 0;
+
+  return openOrders;
+}
 
 export async function createAndDeleteMultipleOrders(numberOfOrders: number) {
   console.log(`=== cancel_orders_batch test ===`);
   await using chainflip = await getChainflipApi();
 
   const keyring = new Keyring({ type: 'sr25519' });
   keyring.setSS58Format(2112);
-  const lpUri = process.env.LP_URI || '//LP_1';
+  const lpUri = process.env.LP_URI || '//LP_3';
```

Comment:
nit: would be nice to define '//LP_3' in one place at the top of the file. Makes it easy to later see what account this test is using and change it.